core/src/main/java/com/microsoft/applicationinsights/telemetry/BaseTelemetry.java

```diff
@@ -21,6 +21,7 @@
 
 package com.microsoft.applicationinsights.telemetry;
 
+import com.microsoft.applicationinsights.agent.internal.common.StringUtils;
```

Comment:
@XiaoliangHuang can we avoid agent import in core classes. I believe there is similar method in core also.

Reply:
updated, the agent dependent on the core, should not refer agent from core.